package.json

```diff
@@ -24,23 +24,29 @@
     "dabapps"
   ],
   "author": "DabApps",
-  "license": "BSD",
```

Comment:
Why are you re-licensing as part of this PR?

Reply:
Because BSD is not a valid license.

Reply:
Looking at the list of licenses NPM considers valid (https://spdx.org/licenses/), you are correct. Changes like this should still be done as a separate PR really!

Would be good to at least add to the readme

Reply:
I've removed this, would be good to open another PR with this change though.

Reply:
"private": true,
fixes it, I was about to add in the other PR but happy to remove it if not needed